package.json

```diff
@@ -1,25 +1,29 @@
 {
-    "name": "storybook-builder-vite-parent",
-    "version": "0.1.13",
-    "description": "An experimental plugin to run and build Storybooks with Vite",
-    "main": "index.js",
-    "author": "",
-    "license": "MIT",
-    "scripts": {
-        "lint": "prettier --check .",
-        "format": "prettier --write ."
-    },
-    "workspaces": [
-        "packages/example-react",
-        "packages/example-svelte",
-        "packages/example-vue",
-        "packages/example-workspaces",
-        "packages/storybook-builder-vite"
-    ],
-    "prettier": {
-        "singleQuote": true
-    },
-    "devDependencies": {
-        "prettier": "^2.3.2"
-    }
+  "name": "storybook-builder-vite-parent",
+  "version": "0.1.13",
+  "description": "An experimental plugin to run and build Storybooks with Vite",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "scripts": {
+    "lint": "cross-env NODE_ENV=production eslint \"packages/*/**/*.{js,jsx,mjs,cjs}\" --fix",
```

Comment:
> If I understand why this is needed we don’t need `NODE_ENV=production` here.

Because, we have a few rules for that disabled for development in ESLint config, see:

https://github.com/eirslett/storybook-builder-vite/blob/34a690bcb6e2ac1f7a8810b47e748574ca5434ca/.eslintrc.js#L14-L16

So, when you use IDE with ESLint plugin, you will not get warnings about using `console` and `debugger`.

But before committing and publishing we need to run linting with enabled rules, for manual code cleanup from breakpoints and and debug output.

Reply:
I think it should be up to developers to configure their editors to ignore those warnings, if that's what they would like.

Reply:
Hmm, I saw this but I've never come across a situation where I've wanted to built this into eslint's config.

Reply:
> I think it should be up to developers to configure their editors to those warnings, if that's what they would like.

@IanVS It's already configured in ESLint rules.

Reply:
> my confusion is that we have lint and lint:ci both are production even though to me lint would be development?

@joshwooding  

* `lint` — used before committing changes, have `--fix` argument;
* `lint-ci` — used before publishing.

They are in production mode, because we need show warning if developer is not deleted `console.*` methods or `debugger` statements.

Reply:
Right, but I think it shouldn't be.  I think they should be warnings no matter the `NODE_ENV`.

Reply:
I think there should always be warnings too. This could cause a lot of confusion.

Reply:
> I think there should always be warnings too.

Instead of focusing on real warnings and errors, you got false positive warnings about `console` and `debugger`:

![image](https://user-images.githubusercontent.com/2821574/148239248-c1d965fa-1560-48df-bfac-bb19cba828cf.png)

It's very annoying when you writing a code.

So, my solution is provide a way for a developer to focus on the real issues, not a ghosts:

![image](https://user-images.githubusercontent.com/2821574/148239116-603e51d5-26ac-488c-87ee-7af89e8a618d.png)

But if I run the `lint` script I can catch the false-positive warning before committing the code:

![image](https://user-images.githubusercontent.com/2821574/148239945-1b04f4b2-7a55-4ce8-a8a1-dcb566d4780c.png)

Reply:
@IanVS @joshwooding how you want to setup `no-console` and `no-debugger` ESLint rules for better developer experience:

* hide false-positive warnings in the development mode, show them in the production mode;
* show them always;
* disable this rules?

Reply:
Resolved in 44fce07.